magefile.go

```diff
@@ -46,6 +47,40 @@ func Build() error {
 	return nil
 }
 
+func Check() error {
```

Comment:
To really make use of the vendored package-registry we probably need to adjust like 42 and 36?

Reply:
I used `go run`.